OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/RegistrationSelectView.java

```diff
@@ -34,14 +34,20 @@ public RegistrationSelectView(RegistrationFormField field, View view) {
         // set prompt
         mInputView.setPrompt(mField.getLabel());
 
-        RegistrationOption defaultOption = null;
+        // Remove json defined default value
```

Comment:
This comment needs more clarification as to why we are doing what we are doing. Something like:
```java
/*
Remove JSON defined default value, which is appropriate for web but not for mobile.
e.g. server sends "--" as the default value for a select box, but on mobile we want
the default value to be the label of select box like Gender, Country etc.
*/
```